examples/stable_diffusion_xl/configs/training/sd_xl_base_finetune_lora.yaml

```diff
@@ -0,0 +1,162 @@
+model:
+    only_save_lora: True
+    target: gm.models.diffusion.DiffusionEngine
+    params:
+        scale_factor: 0.13025
+        disable_first_stage_amp: False
+
+        denoiser_config:
+            target: gm.modules.diffusionmodules.denoiser.DiscreteDenoiser
+            params:
+                num_idx: 1000
+
+                weighting_config:
+                    target: gm.modules.diffusionmodules.denoiser_weighting.EpsWeighting
+                scaling_config:
+                    target: gm.modules.diffusionmodules.denoiser_scaling.EpsScaling
+                discretization_config:
+                    target: gm.modules.diffusionmodules.discretizer.LegacyDDPMDiscretization
+
+        network_config:
+            target: gm.modules.diffusionmodules.openaimodel.UNetModel_lora
+            params:
+                lora_dim: 4                 # for LoRA
+                lora_alpha: 32              # for LoRA
```

Comment:
why is alpha so large? typically it is 1 referring to diffusers or cloneofsimo/lora.

Reply:
32 is reference from gpt-2 lora. The experiment seems to have similar effects with alpha 4 and 32, currently modified to the same configuration as in diffuser, alpha -> 4(scale 1).